components/playlist/browser/resources/components/playerSeeker.tsx

```diff
@@ -200,7 +200,7 @@ export default function PlayerSeeker ({ videoElement }: Props) {
 
   React.useEffect(() => {
     if (!videoElement) {
-      console.error(`Video element doesn't exist`)
+      // Can be null on start up.
```

Comment:
Ah yeah :/ I missed that last time sorry :laugh

Reply:
No worries at all 😄